data-pipeline/src/data_pipeline/data_types/gene.py

```diff
@@ -226,6 +226,12 @@ def import_hgnc(path):
     return ds
 
 
+def prepare_table_for_release(genes_path):
+    ds = hl.import_table(genes_path)
+    ds = ds.select_globals()
```

Comment:
it might be helpful to retain the MANE select version on this table since it contains the gene model information